release-notes/5.0/api-diff/netcoreapp3.1/.Net/standalone/5.0-standalone_System.Threading.AccessControl.md

```diff
@@ -0,0 +1,18 @@
+# System.Threading.AccessControl
+
+``` diff
+ {
+     namespace System.Threading {
++        public static class EventWaitHandleAcl {
++            public static EventWaitHandle Create(bool initialState, EventResetMode mode, string name, out bool createdNew, EventWaitHandleSecurity eventSecurity);
++        }
++        public static class MutexAcl {
++            public static Mutex Create(bool initiallyOwned, string name, out bool createdNew, MutexSecurity mutexSecurity);
++        }
++        public static class SemaphoreAcl {
++            public static Semaphore Create(int initialCount, int maximumCount, string name, out bool createdNew, SemaphoreSecurity semaphoreSecurity);
```

Comment:
In case we want to show nullability in these diffs, nullability is also missing in some parameters of these 3 methods.

Reply:
we avoid nullability changes in api diff